apps/dsp-app/src/app/project/ontology/ontology.component.ts

```diff
@@ -145,6 +145,10 @@ export class OntologyComponent implements OnInit {
     // disable content on small devices
     disableContent = false;
 
+    // route to classes view
+    classesLink = `../${RouteConstants.classes}`;
```

Comment:
instead of creating multiple variables, you can just pass routeConstants to the view: 
routeConstants = RouteConstants;